### PR TITLE
cookiecutter: todo additions and entry point fix

### LINKED ISSUE #35: cookiecutter: add notes to examples files about removal if unused
Add TODO notes in example files that some things should be removed if not used.

### PR DESCRIPTION
**Pull request by lnielsen**
* Adds extra todo items after bootstrapping. (closes #35)

* Adds entry points groups to setup.py. (addresses #31)

Signed-off-by: Lars Holm Nielsen <lars.holm.nielsen@cern.ch>

### CODE REVIEW

File: {{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/views.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 from __future__ import absolute_import, print_function
 
+# TODO: This is an example file. Remove it if you do not need it including
```
Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/views.py]: isort suggests to move it after all imports

File: {{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/views.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {% include 'misc/header.py' %}
 """{{ cookiecutter.description }}"""
 
+# TODO: This is an example file. Remove it if you do not need it including
```
Comment (nharraud) [inline {{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/views.py]: minor detail, s/not need it including/not need it, including/
adding a coma

**Comment by jirikuncar** [comment]
:shipit:

**Comment by nharraud** [comment]
@lnielsen @jirikuncar I don't have the write permissions on this repository

**Pull request merged**